packages/botonic-core/src/hubtype-service.js

```diff
@@ -1,9 +1,18 @@
 import axios from 'axios'
 import Pusher from 'pusher-js'
 
-const PUSHER_KEY = process.env.WEBCHAT_PUSHER_KEY || '434ca667c8e6cb3f641c'
-const HUBTYPE_API_URL = process.env.HUBTYPE_API_URL || 'https://api.hubtype.com'
-
+const PUSHER_KEY =
+  (typeof WEBCHAT_PUSHER_KEY !== 'undefined'
+    ? // eslint-disable-next-line no-undef
+      WEBCHAT_PUSHER_KEY
+    : typeof process !== 'undefined' && process.env.WEBCHAT_PUSHER_KEY) ||
+  '434ca667c8e6cb3f641c'
+const HUBTYPE_URL =
```

Comment:
why did you rename it? I think it's less error prone to keep the same name

Reply:
I think so, but i can't have two symbols with the same name (one that references the webpack env variables, and one for declaring the variable

Reply:
now refactored, using `getWebpackEnvVar`